src/main/java/net/dv8tion/jda/core/entities/impl/GuildImpl.java

```diff
@@ -676,4 +671,37 @@ public String toString()
     {
         return "G:" + getName() + '(' + getId() + ')';
     }
+
+    @Override
+    public RestAction<List<Invite>> getInvites()
+    {
+        if (!this.getSelfMember().hasPermission(Permission.MANAGE_SERVER))
+            throw new PermissionException(Permission.MANAGE_SERVER);
+
+        final Route.CompiledRoute route = Route.Invites.GET_GUILD_INVITES.compile(getId());
+
+        return new RestAction<List<Invite>>(api, route, null)
+        {
+            @Override
+            protected void handleResponse(final Response response, final Request request)
+            {
+                if (response.isOk())
+                {
+                    EntityBuilder entityBuilder = EntityBuilder.get(this.api);
+                    JSONArray array = response.getArray();
+                    List<Invite> invites = new ArrayList<>(array.length());
+                    for (int i = 0; i < array.length(); i++)
+                    {
+                        invites.add(entityBuilder.createInvite(array.getJSONObject(i)));
+                    }
+                    request.onSuccess(invites);
+                }
+                else
+                {
+                    request.onFailure(response);
+                }
```

Comment:
why exactly is this better? for very long/complex methods I agree it's easier readable, but not for simple ones like this